packages/kit/package.json

```diff
@@ -62,13 +62,13 @@
 		"lint": "eslint --ignore-path .gitignore --ignore-pattern \"src/packaging/test/**\" \"{src,test}/**/*.{ts,mjs,js,svelte}\" && npm run check-format",
 		"check": "tsc && npm run check:integration && npm run check:prerendering",
 		"check:integration": "npm run check:integration:amp && npm run check:integration:basics && npm run check:integration:options && npm run check:integration:options-2",
-		"check:integration:amp": "cd test/apps/amp && pnpm check",
-		"check:integration:basics": "cd test/apps/basics && pnpm check",
-		"check:integration:options": "cd test/apps/options && pnpm check",
-		"check:integration:options-2": "cd test/apps/options-2 && pnpm check",
+		"check:integration:amp": "cd test/apps/amp && pnpm build && pnpm check",
+		"check:integration:basics": "cd test/apps/basics && pnpm build && pnpm check",
+		"check:integration:options": "cd test/apps/options && pnpm build && pnpm check",
+		"check:integration:options-2": "cd test/apps/options-2 && pnpm build && pnpm check",
 		"check:prerendering": "npm run check:prerendering:basics && npm run check:prerendering:options",
-		"check:prerendering:basics": "cd test/prerendering/basics && pnpm check",
-		"check:prerendering:options": "cd test/prerendering/options && pnpm check",
+		"check:prerendering:basics": "cd test/prerendering/basics && pnpm build && pnpm check",
+		"check:prerendering:options": "cd test/prerendering/options && pnpm build && pnpm check",
```

Comment:
we don't want to build these projects every time we check them, it'll slow things right down. in CI we do `pnpm -r build` before running the checks. if we really want to eliminate the chance of checking prematurely in _all_ cases then we should use `svelte-kit sync` (#4182)

Reply:
`pnpm -r build` only builds the code. It doesn't build these test projects. I thought it'd be more of a negative impact to build these during `pnpm build`

Yeah, my plan was to switch these to `sync` as soon as it's available

Reply:
> It doesn't build these test projects

it does!

Reply:
Ah, you're right. I was thinking of `pnpm build` which has a filter on it.

Anyway, will we just switch these to `sync` when it's available? I'm not sure it matters what happens in the meantime since it will be in this state for only a short time. I can remove `pnpm build` here if you want and then we will just add it back as `pnpm sync`?

Reply:
Ok. Can we just leave this as `build` then and have your PR update it to `sync`? If I try to remove it then I'm going to have to update the `.github/workflows` just for you to remove my changes in your PR, which I think would be more work for both of us

Reply:
updated to use `sync`